packages/amplify-e2e-core/src/init/amplifyPush.ts

```diff
@@ -203,6 +203,16 @@ export const amplifyPushAuth = (cwd: string, testingWithLatestCodebase = false):
     .wait(/.*/)
     .runAsync();
 
+/**
+ * Function to test amplify push
+ */
+export const amplifyPushAuthLegacy = (cwd: string, testingWithLatestCodebase = false): Promise<void> =>
```

Comment:
Instead of `Legacy` please use the latest version it applies to like `amplifyPushV10` or something like that.

Reply:
fixed. I haven't touch existing `Legacy` methods to keep parity with dev.